### PR TITLE
[Edit] CSS Box Sizing: Add media

### LINKED ISSUE #3646: [Edit] CSS [Edit] CSS: Box Sizing
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

[PLEASE DO NOT WORK ON THIS ISSUE UNTIL YOU HAVE BEEN ASSIGNED] We would like an edit to the CSS Box Sizing concept. The entry lives in docs/content/css/concepts/box-sizing/box-sizing.md

Please refer to the [concept entry templates](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add supplemental media (preferred .png or .gif, <1MB).
2) Media (images, gifs) should accurately represent the existing code blocks in the example sections and fit in-line. ex: [See last image here](https://www.codecademy.com/resources/docs/html/attributes/title)

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Jamesllllllllll**
### Description

- Added an image to show an example of the `box-sizing` property. 
- Re-wrote the metadata description to be a single sentence per the [Style Guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#metadata) (`Description`: should be short, typically one sentence.)
- Removed opinionated sentence about this property being advanced. This is taught in the Codecademy HTML & CSS courses. New web developers should know this early on, in my opinion.

Closes #3646 

### Type of Change

- Editing an existing entry: Adding media with description, and editing current entry.

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/css/concepts/box-sizing/box-sizing.md [hunk 1]
```diff
@@ -45,3 +45,12 @@ Setting a `div` elements `box-sizing` to `content-box`.
   box-sizing: content-box;
 }
 ```
+## Example 3
+
+The following image has two `div` containers with a width of 300px, border of 1px and padding of 20px. Inside each container there is a `div` with a width of 100%.
```
Comment (dakshdeepHERE) [inline content/css/concepts/box-sizing/box-sizing.md:50]:
```suggestion
The following image has two `div` containers with a width of 300px, a border of 1px, and a padding of 20px. Inside each container, there is a `div` with a width of 100%.
```

File: content/css/concepts/box-sizing/box-sizing.md [hunk 2]
```diff
@@ -45,3 +45,12 @@ Setting a `div` elements `box-sizing` to `content-box`.
...
+
+The first container `div` is set to `border-box`. Its total width including the black border, tan padding and blue `div` is 300px.
+
+The second container `div` is set to `content-box`. The width of the blue content `div` is 300px. The total width of the container `div` with padding and border and content `div` is 342px.
```
Comment (dakshdeepHERE) [inline content/css/concepts/box-sizing/box-sizing.md:54]:
```suggestion
The first container `div` is set to `border-box`. Its total width including the black border, tan padding, and blue `div` is 300px.

The second container `div` is set to `content-box`. The width of the blue content `div` is 300px. The total width of the container `div` with padding and border and content `div` is 342px.
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Few changes from my side @Jamesllllllllll.

**Comment by dakshdeepHERE** [comment]
@Jamesllllllllll this PR has been closed by you. If you want you can re-push the PR.

**Comment by Jamesllllllllll** [comment]
@dakshdeepHERE whoops! My mistake. Thank you, I will do that.

**Pull request closed without merging**